core/driver/sources/src/main/scala/org/virtuslab/ideprobe/IdeProbeFixture.scala

```diff
@@ -4,18 +4,22 @@ import java.util.concurrent.{SynchronousQueue, ThreadPoolExecutor, TimeUnit}
 import scala.concurrent.ExecutionContext
 import scala.util.Try
 
-trait IdeProbeFixture {
-  protected implicit val ec: ExecutionContext = {
+object IdeProbeFixture {
+  def defaultEc(): ExecutionContext = {
```

Comment:
Nit:
```suggestion
  def defaultEC(): ExecutionContext = {
```
as per Java conventions for <3 letter acronyms

Reply:
Done